SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,55 @@
+header:
+  schema-version: 1.0.0
+  expiration-date: '2024-12-16'
+  project-url: 
+project-lifecycle:
+  bug-fixes-only: false
+  core-maintainers:
+  - ellistarn
+  - jonathan-innis
+  - tzneal
+  - bwagner5
+  - njtran
+  status: active
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  automated-tools-list:
+    - automated-tool: dependabot
+      action: allowed
+      path:
+        - /
+distribution-points:
+  - https://github.com/kubernetes-sigs/karpenter/blob/main/RELEASE.md
```

Comment:
This isn't the distribution point. I'd think that a more appropriate URL might be: https://pkg.go.dev/sigs.k8s.io/karpenter since this is distributed as a Go package